utils/generate-fmtargs.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 # Outputs the generated part of src/fmtargs.h
-MAX_ARGS = 120
+MAX_ARGS = 122
```

Comment:
```suggestion
MAX_ARGS = 200
```
When I wrote this there was a lot of headroom, I'm kind of surprised we hit this so quickly. Let's bump it up higher so that it's awhile before we need to udpate it again.

Reply:
Done.